metadata.yaml

```diff
@@ -43,9 +43,9 @@ requires:
   redis:
     interface: redis
     limit: 1
-  db:
-    interface: pgsql
-    limit: 1
+  database:
+      interface: postgresql_client
+      limit: 1
```

Comment:
Any reason for the extra spaces before `interface` and `limit` here? Would be good to consistent with existing entries

Reply:
Fixed